docs/tutorials/web/excel-addin-bert-js.md

```diff
@@ -0,0 +1,346 @@
+---
+title: Custom Excel Functions for BERT Tasks in JavaScript
+description:  Custom Excel Functions for BERT Tasks in JavaScript
+parent: Deploy on web
+grand_parent: Tutorials
+has_children: false
+nav_order: 2
+---
+
+# ONNX Runtime Custom Excel Functions for BERT NLP Tasks in JavaScript
+{: .no_toc }
+
+In this tutorial we will look at how we can create custom excel functions (`ORT.Sentiment()` and `ORT.Question()`) to implement BERT NLP models with ONNX Runtime Web to enable deep learning in spreadsheet tasks. The inference happen locally in the browser with excel on the web. 
+
+<div class="embed-container">
+  <iframe
+      src="https://www.youtube.com/embed/wuSxWGY_Sjg"
+      width="560" height="315"
+      frameborder="0"
+      allowfullscreen="true">
+  </iframe>
+</div>
+
+## Contents
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+## Prerequisites
+
+- [See the prerequisites from this tutorial](https://learn.microsoft.com/en-us/office/dev/add-ins/tutorials/excel-tutorial-create-custom-functions?source=recommendations&tabs=excel-windows#prerequisites)
```

Comment:
Would it be clearer to list the pre-requisites here? As far as I can see there are only 3

Reply:
I do not think we should do this. The prerequisite is to follow the steps to create the sample project and there is no need to duplicate that. I will leave as is.

Reply:
Added prerequisites from linked tutorial